### PR TITLE
Add improved Description for PyPI

### LINKED ISSUE #246: Improve Project Description page with README on PyPI
https://pypi.org/project/sphinxcontrib-jupyter/

### PR DESCRIPTION
**Pull request by mmcky**
fixes #246

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -4,9 +4,36 @@
 
 VERSION = 'v0.5.0'
 
-long_desc = '''
-This package contains the Jupyter Sphinx extension.
-'''
+LONG_DESCRIPTION = """
+This package contains a `Sphinx <http://www.sphinx-doc.org/en/master/>`_ for compiling
```
Comment (slel) [inline setup.py]: missing the word "extension" in "contains a Sphinx extension"

Comment (mmcky) [reply setup.py]: thanks @slel fixed.

**Comment by mmcky** [comment]
not sure why travis is failing here -- `dask distributed` is installed for `python37`

**Comment by mmcky** [comment]
moved to using `pip` for the `travis` environment.

**Pull request merged**